### ISSUE TITLE
Make Choices Accessible

### ISSUE THREAD

**Issue by tkoleary** [Choices-js/Choices#793]
It's great that Choices has no jQuery dependency. What would be amazing is if it was fully WCAG compliant. AFAIK neither Select2 and Chosen are WCAG compliant which would make Choices the first. ;)

**Comment by jshjohnson** [comment]
I've attempted to make Choices as accessible as I can - if you can see ways it can be improved, I'm all ears :)

**Comment by brianteeman** [comment]
An issue I discovered is with the use of `role=treeitem`

The parent has a role of

`<div class="choices__list" aria-multiselectable="true" role="listbox">`

The children have a role of treeitem

`<div id="choices--choices-multiple-groups-item-choice-17" class="choices__item choices__item--choice choices__item--selectable is-highlighted" role="treeitem" data-choice="" data-id="17" data-value="Montreal" data-select-text="Press to select" data-choice-selectable="" aria-selected="true">Montreal</div>`

A treeitem role is assigned to the element which is an option within the tree role element.

So by my understanding of this the parent should be 
`<div class="choices__list" aria-multiselectable="true" role="tree">`

A useful way to find these issues is by using the google chrome lens tool https://chrome.google.com/webstore/detail/chromelens/idikgljglpfilbhaboonnpnnincjhjkd?hl=en

**Comment by Xon** [comment]
Please try the latest version on https://choices-js.github.io/Choices/

This has had a few changes for aria attributes, but testing with accessibility tools is not something I have experience with, which makes validating these are good changes challanging.
